### DISCUSSION TITLE
Is there a way to create a Micro PDF417 barcode?

### DISCUSSION THREAD

**Discussion by Kizuto3** [micjahn/ZXing.Net#450] [Q&A] [answered]
I've noticed that it is possible to create a Macro PDF417, but I didn't find any way to create a Micro PDF417. Does ZXing support Micro PDF417? If not, is there any plans to implement it?

**Comment by micjahn** [comment, marked as answer]
No, it doesn't support it and there are no plans to implement it. Sorry.
